src/plugin/index.js

```diff
@@ -6,7 +6,9 @@ const importCwd = require("import-cwd"); // used to get the users project detail
 const pkg = importCwd("./package.json");
 
 function Jarvis(options) {
-  // TOOD: add options for port, etc..
+  this.options = {
+    port: options && options.port || 1337 // if no port specified fall back to 1337
+  }
```

Comment:
And maybe you can add a semicolon here after merging to keep the code-style, I don't use them so I forgot to add one here, sorry

Reply:
I guess I'll have prettier on pre-commit hook but now let's go with default prettier settings from your editor